users/gods_group_membership.tf

```diff
@@ -0,0 +1,11 @@
+# Put our admin IAM users in the Terraform backend access and user
```

Comment:
You missed changing this "admin" to "god-like".

Reply:
Remedied in 8397be1.